docs/postcss.config.js

```diff
@@ -13,17 +12,6 @@ module.exports = {
       features: {
         'nesting-rules': true
       },
-      importFrom: [
-        {
-          customMedia: {
-            '--breakpoint-x-small': `(width >= ${core.breakpoints.xSmall})`,
-            '--breakpoint-small': `(width >= ${core.breakpoints.small})`,
-            '--breakpoint-medium': `(width >= ${core.breakpoints.medium})`,
-            '--breakpoint-large': `(width >= ${core.breakpoints.large})`,
-            '--breakpoint-xLarge': `(width >= ${core.breakpoints.xLarge})`
-          }
-        }
-      ],
```

Comment:
I'm seeing some issues with the docs layout - it seems to be stuck in the mobile layout. i believe this omission/deletion is the cause.

Reply:
Yes, at one point in development, this was a problem, but I inlined the breakpoints in the media queries, so the values are there.

I'd probably go to docs/ and:

```
rm -rf .cache dist
yarn
yarn start:dev
```

Reply:
I tried those steps but still have the issue